syntaxes/stellaris_yaml_injection.json

```diff
@@ -39,120 +39,120 @@
       ]
     },
     "bracket-command": {
-      "match": "(?<!\\[)\\[[^\\s\\[\\]\\n]+\\]",
+      "match": "\\[[^\\s\\n\\[\\]\\$]+\\]",
       "name": "bracket-command.stellaris.yml"
     },
     "loc-key": {
-      "match": "\\$[^\\s\\$\\n]+\\$",
+      "match": "\\$[^\\s\\n\\$]+\\$",
       "name": "loc-key.stellaris.yml"
     },
     "icon-code": {
-      "match": "\\£[^\\s\\£\\n]+\\£",
+      "match": "\\£[^\\s\\n\\£]+\\£",
       "name": "keyword.icon-code.stellaris.yml"
     },
     "color-code-w": {
-      "begin": "§W(?=[^\\n]*§!)",
-      "end": "§!",
+      "begin": "§W(?=[^\\n]*(§!|(?=\"\\n)))",
```

Comment:
I don't think we need this complex expression anymore. The way it is now it basically a `match` but it is better to use a simple `begin` and `end` and let VSCode figure out the magic :D If we use a `match` we would have to find a way to account for nested color codes

Reply:
Seems to be working as expected now